pkg/resolution/resource/crd_resource.go

```diff
@@ -122,7 +123,11 @@ func appendOwnerReference(rr *v1beta1.ResolutionRequest, req Request) {
 }
 
 func ownerRefsAreEqual(a, b metav1.OwnerReference) bool {
-	return a.APIVersion == b.APIVersion && a.Kind == b.Kind && a.Name == b.Name && a.UID == b.UID && a.Controller == b.Controller
+	// pointers values cannot be directly compared.
+	if !reflect.DeepEqual(a.Controller, b.Controller) {
```

Comment:
Controller is just a pointer to a boolean right? can you dereference the pointer? I'm just curious here, I think what you have works.

Reply:
I just don't want to write a long if statement to check conditions.
If you think `reflect.DeepEqual` is too heavy, I'll switch back to the regular way. 😁